README.md

```diff
@@ -22,6 +22,7 @@ If you cannot provide a valid website for your company your PR will not be accep
 
 ## A - C
 - [Allied Netlink Technology Pvt. Ltd.](http://ant.com.np/) | Putalisadak, Kathmandu | Internet Service Provider (ISP)
+- [Ambarkaar Software Pvt. Ltd.](http://www.http://compliancequest.com//) | Bakhundol, Lalitpur | Ambarkaar is a subsidiary company of ComplianceQuest. We develope Enterprise Quality Management System in Salesforce.   
```

Comment:
Hi, Can we remove all full stops from the description? Your URL is also not correct.

You can go here and edit the file again: https://github.com/Prashan049/tech-companies-in-nepal/tree/patch-1